marklogic-data-hub/src/test/java/com/marklogic/hub/hubcentral/ApplyDownloadedZipToProjectTest.java

```diff
@@ -36,7 +36,7 @@
      */
     @Test
     void deleteOneEntityThenDownloadAndApply() {
-        installProjectInFolder("test-projects/all-artifacts");
+        installProjectInFolder("test-projects/download-artifacts");
```

Comment:
There are a few other tests that depend on this folder - let's hold off on renaming this, as it's adding to the number of files modified in this PR.

Reply:
Since we are deleting the entity model which is referenced by a couple of mapping artifacts in the `ApplyDownloadedZipToProjectTest` it used to fail based on the new changes in this PR.

I created a new project called `download-artifacts` based on the `all-artifacts`(not touching it since other tests depend on it) project, the difference here is that I've removed the steps that referenced the Entity that was being deleted so the tests can pass.